data/json/npcs/cabin chemist/chemist missions.json

```diff
@@ -0,0 +1,108 @@
+[
+{
+    "id": "MISSION_CABIN_CHEMIST_GET_analytical_set_basic",
+    "type": "mission_definition",
+    "name": "Find A Basic Analysis Kit",
+    "description": "Obtain a basic laboratory analysis kit for the chemist.",
+    "goal": "MGOAL_FIND_ITEM",
+	"item": "analytical_set_basic",
+	"count": 1,
+    "difficulty": 2,
+    "value": 3000,
+	"origins": [ "ORIGIN_OPENER_NPC" ],       
+        "end": {
+           "effect": [ { "npc_add_var": "mass_chem_tradeable", "type": "dialogue", "context": "trade", "value": "yes" } ]
+               },    
+    "followup": "MISSION_CABIN_CHEMIST_GET_100_chem_muriatic_acid",   
+	"has_generic_rewards": true,
+	"dialogue": {
+      "describe": "I need a basic laboratory analysis kit.",
+	  "offer": "I've checked the stock here at this house, and it turns out there's a few pieces of equipment I need. If you could get me a basic laboratory analysis kit, I'd be grateful. Heck, I might take some specialty orders for you.",

```

Comment:
Please add two spaces instead of one after periods between sentences. Here and below.

Reply:
> Please add two spaces instead of one after periods between sentences. Here and below.

That has been done.